### PR TITLE
Issue 144

### PR DESCRIPTION
**Pull request by bhuvan-rk**
I've added text to the bashrc, virtualenv and Kibana sections of HACKING.md. Please review and comment. 

Thanks
Bhuvana

### CODE REVIEW

**Comment by jxstanford** [comment]
:+1:

File: docs/HACKING.md [hunk 1]
```diff
@@ -55,10 +55,18 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
     export PROJECT_HOME=$HOME/devel
     source /usr/local/bin/virtualenvwrapper.sh
 
+Wherever you add these flags, ensure to source it on all your open terminal windows. On most MAC OS's the ".bashrc" file does not exist in the "~/.bashrc" path. You might be able to find equivalent files like "/etc/bash_profile" or "/etc/profile". You can also create your own "/etc/bashrc" file. If you do create your own file, add these lines to your bash_profile: 
+
+    if [ -f ~/.bashrc ]; then
+        . ~/.bashrc
+    fi 
+
```
Comment (stugots) [inline docs/HACKING.md]: I've got a ~/.bashrc and it works fine.

File: docs/HACKING.md [hunk 2]
```diff
@@ -55,10 +55,18 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
...
 Open a new terminal window and confirm that these environment variables have been set.  Once satisfied, move on to creating the virtualenv:
 
     $ mkvirtualenv -a $PROJECT_HOME/goldstone-server goldstone-server
 
+If mkvirtualenv command fails, check to see if WORKON_HOME and other environmentvariables you exported earlier are set correctly. In some cases, virtualenvwrapper.sh can be located in /usr/bin instead of /usr/local/bin. Change the variables correspondingly in your settings and source them again.
```
Comment (stugots) [inline docs/HACKING.md]: environment space variables

**Comment by stugots** [comment]
:+1: after you consider my comments.

File: docs/HACKING.md (hunk 2 above)
Comment (bhuvan-rk) [inline docs/HACKING.md]: Good catch, I'm adding some more text to this document. I'll add environment <space> variables  with that.

File: docs/HACKING.md (hunk 1 above)
Comment (bhuvan-rk) [inline docs/HACKING.md]: I did not have one. All of my settings are sourced from /etc/bash_profile. Some very long stackoverflow threads exist explaining the reason why. 

Also note the words "On most MAC OS's" :)

Comment (itlinux) [inline docs/HACKING.md]: I have mine .bash_profile like this on my mac just to show a few additional things you may need :) 


if [ -f /opt/local/etc/profile.d/bash_completion.sh ]; then
    . /opt/local/etc/profile.d/bash_completion.sh
fi

p2=/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/
PATH=/usr/local/bin:${PATH/:\/usr\/local\/bin}:/usr/texbin:${p2}

if [ -f ~/.bashrc ]; then
          source ~/.bashrc
        else
       echo "No .bashrc file"
fi

**Comment by bhuvan-rk** [comment]
Good catch, I'm adding some more text to this document. I'll add environment "space" variables with that.

**Comment by bhuvan-rk** [comment]
I did not have a bashrc. All of my settings are sourced from /etc/bash_profile. Some very long stackoverflow threads exist explaining the reason why.

Also note the words "On most MAC OS's" :)

**Pull request merged**